src/pybind/mgr/dashboard_v2/.pylintrc

```diff
@@ -131,7 +131,11 @@ disable=print-statement,
         invalid-name,
         no-self-use,
         too-few-public-methods,
-        no-member
+        no-member,
+        bad-continuation,
```

Comment:
this will make the code non-uniform across files, do you really need to disable this?

Reply:
It forces a specific continuation style that was not enforced before. I've created https://tracker.openattic.org/browse/OP-3101